src/ir_Argo.cpp

```diff
@@ -505,4 +1748,98 @@ bool IRrecv::decodeArgo(decode_results *results, uint16_t offset,
   // is a union data type.
   return true;
 }
+
+/// Decode the supplied Argo message (WREM3).
+/// Status: Confirmed working w/ Argo 13 ECO (WREM-3)
+/// @param[in,out] results Ptr to the data to decode & where to store the decode
+///   result.
+/// @param[in] offset The starting index to use when attempting to decode the
+///   raw data. Typically/Defaults to kStartOffset.
+/// @param[in] nbits The number of data bits to expect.
+/// @param[in] strict Flag indicating if we should perform strict matching.
+/// @return A boolean. True if it can decode it, false if it can't.
+/// @note This decoder is separate from @c decodeArgo to maintain backwards
+///       compatibility. Contrary to WREM2, this expects a footer and gap!
+bool IRrecv::decodeArgoWREM3(decode_results *results, uint16_t offset,
+                             const uint16_t nbits,
+                             const bool strict) {
+  if (strict
+      && nbits != kArgo3AcControlStateLength * 8
+      && nbits != kArgo3ConfigStateLength * 8
+      && nbits != kArgo3iFeelReportStateLength * 8
+      && nbits != kArgo3TimerStateLength * 8) {
+    return false;
+  }
+
+  const uint16_t kArgoOverhead = 3;
+  uint16_t bytesRead = matchGeneric(results->rawbuf + offset, results->state,
+                  results->rawlen - offset, nbits,
+                  kArgoHdrMark, kArgoHdrSpace,
+                  kArgoBitMark, kArgoOneSpace,
+                  kArgoBitMark, kArgoZeroSpace,
+                  kArgoBitMark, kArgoGap,  // difference vs decodeArgo
+                  true, _tolerance, kArgoOverhead,
```

Comment:
Thanks for pointing that out! This now seems to me to be a bad refactor or something...And as far as the "am I sure" part... nah, not at all. I only know what I inferred about it by reading the code, so might be missing the point entirely.

----

I remember fiddling with all these parameters at some point, as I had a hit-and-miss real-life decode results, but for the life of me, I can't recall why I have changed this particular value. 

I'll revert it to `kMarkExcess` for now (UTs are passing, and these are real captures, so it can't be that bad :) ). 
If I later recall why I have changed it, or I see issues with it in practice, will raise as a separate PR.

Reply:
As it's working with `3`, you can just set (try) the excess to `0`.

Reply:
Done